scripts/build_nim.sh

```diff
@@ -26,7 +26,8 @@ CI_CACHE="$4"
 [[ -z "$V" ]] && V=0
 [[ -z "$CC" ]] && CC="gcc"
 # to build csources in parallel, set MAKE="make -jN"
-[[ -z "$MAKE" ]] && MAKE="make"
+# the default is 3 processes which is better than 1 but still not too taxing
+[[ -z "$MAKE" ]] && MAKE="make -j3"
```

Comment:
Where is this supposed to kick in? We set the "MAKE" env var in the "build-nim" target: https://github.com/status-im/nimbus-build-system/blob/92e5042667b747d22106f085eaa9b5e9766ba474/makefiles/targets.mk#L60

and in CI, when using this script directly: https://github.com/status-im/nim-libp2p/blob/7b5259dbc713851f5fd14d2024fedf2cee4f95c7/azure-pipelines.yml#L102